docs/utilities/parser.md

```diff
@@ -249,3 +249,51 @@ The package `@types/aws-lambda` is a popular project that contains type definiti
 Powertools parser utility also bring AWS Lambda event types based on the built-in schema definitions.
 
 We recommend to use the types provided by the parser utility. If you encounter any issues or have any feedback, please [submit an issue](https://github.com/aws-powertools/powertools-lambda-typescript/issues/new/choose).
+
+## Testing your code
+
+When testing your handler with **parser decorator** you need to use double assetion to bypass TypeScript type checking in your tests.
```

Comment:
I think we could link to the section in the docs that talk about it on the words `parser decorator`.